.github/workflows/update-spellcheck-binaries.yml

```diff
@@ -0,0 +1,71 @@
+name: Update node-spellchecker binaries
+on:
+  push:
+    paths:
+      - 'lib/bin/node-spellchecker/**'
+jobs:
+  windows-binaries:
+    name: Windows Binaries
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: npm ci
+        working-directory: lib/bin/node-spellchecker
+      - run: ./build-windows.cmd
+        shell: cmd
+        working-directory: lib/bin/node-spellchecker
+      - run: git pull --no-edit
+        shell: bash
+      - uses: matheusalbino/git-commit@v1.1.0
```

Comment:
Windows need to use a different github action to commit changes back, it's a little more cumbersome than the one from stefanzweifel  (needs the branch name extracted manually and more config)